commerce_coordinator/apps/titan/pipeline.py

```diff
@@ -43,7 +43,7 @@ class GetTitanPayment(PipelineStep):
     Adds Titan's payment in payment data list.
     """
 
-    def run_filter(self, edx_lms_user_id, payment_number=None):  # pylint: disable=arguments-differ
+    def run_filter(self, edx_lms_user_id, payment_number=None, **kwargs):  # pylint: disable=arguments-differ
```

Comment:
This `kwargs` will accommodate all remaining params we pass from the filter but this PipelineStep does not necessarily need them.